Library/Homebrew/software_spec.rb

```diff
@@ -336,20 +336,18 @@ def checksum_for(tag)
   end
 
   def checksums
-    checksums = {}
-    os_versions = collector.keys
-    os_versions.map! do |macos|
+    tags = collector.keys.sort_by do |tag|
```

Comment:
Yes, it does maintain the current ordering, which is reverse order by version number.